modules/release-notes/con-relnotes-known-issues.adoc

```diff
@@ -22,3 +22,12 @@ If you deploy {product-short} without a proxy, there is the potential for IP add
 * Prevent collection of IP addresses by configuring a reverse proxy in front of {product-short}.
 //link:{LinkRHIDPIssue}RHIDP-2766[RHIDP-2766]
 --
+
+Rollbacks are not supported::
++
+--
+If you deploy {product-short} using either Helm or the Operator, bear in mind that rollbacks are not supported.
+It is strongly advised to take regular backups of your {product-short} PostgreSQL database, to be able to restore the application in case of issues.
+In the future, we plan to provide guidance on how to perform rollbacks from existing backups.
```

Comment:
```suggestion
We plan to provide guidance on how to perform rollbacks from existing backups in the future
```
Is it ok to "promise" the delivery of future plans here? Maybe we take this out?

Reply:
@linfraze I think it's ok to say "In future versions of {product-short}, we plan..."

Reply:
I thought that purposely documenting something as a known issue means that we implicitly hope in the future to fix it (unless we document a workaround) since we're aware of it :)

Reply:
I've rephrased it as `In future versions of {product-short}, ...`